src/SFA.DAS.EAS.Web/ViewModels/TransferCommitmentViewModel.cs

```diff
@@ -16,5 +18,9 @@ public TransferCommitmentViewModel()
         public decimal TotalCost { get; set; }
         public List<TransferCourseSummaryViewModel> TrainingList { get; set; }
         public string TransferApprovalStatus { get; set; }
+        public string TransferApprovalSetBy { get; set; }
+        public DateTime? TransferApprovalSetOn { get; set; }
+        public bool PendingApproval => TransferApprovalStatus == "Pending";
```

Comment:
Not a fan of 'magic strings' (for want of a better term!). Better to use an enum in this instance for TransferApprovalStatus. Some reasons are listed in this article... https://daveaglick.com/posts/eliminating-magic-strings, although there are other reasons also.

Reply:
done